notebooks/failure-type-classification/failure_type_functions.ipynb

```diff
@@ -84,7 +84,7 @@
     "            aws_secret_access_key=self.aws_secret_access_key,\n",
```

Comment:
<p>Replace "function" with "class" in the markdown. Also, if we add a sentence or two for the class docstring, it'll be more descriptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>